src/components/Switch/Switch.testrender.tsx

```diff
@@ -1,33 +1,21 @@
 import { createSignal } from 'solid-js';
 
-import artisan from '@cubeartisan/cubeartisan/components/factory';
-import { VStack } from '@cubeartisan/cubeartisan/components/Stack';
 import { Switch } from '@cubeartisan/cubeartisan/components/Switch/Switch';
-import * as styles from '@cubeartisan/cubeartisan/components/Switch/Switch.css';
+import * as styles from '@cubeartisan/cubeartisan/components/Switch/Switch.testrender.css';
 
 export const TestSwitch = () => {
   const [checked, setChecked] = createSignal(false);
 
   return (
-    <VStack atoms={{ gap: 4 }} recipe={{ align: 'center', justify: 'center' }}>
+    <>
       <Switch isChecked={checked()} onCheckedChange={setChecked}>
         <Switch.Input />
         <Switch.Label class={styles.switchLabel}>Test Switch</Switch.Label>
```

Comment:
CI is showing errors here

Reply:
Not sure exactly what the error is here. I cleaned up the class assignments and moved them primarily into the base component but this might need to be revisited